src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```
@@ -1623,4 +1647,21 @@ export class PatientViewPageStore {
         },
         []
     );
+
+    @computed get referenceGenomeBuild() {
+        return currentReferenceGenomeBuildUrl(this.studies.result);
```

you should throw an error here if this.studies.isComplete === false.